sklearn/metrics/tests/test_score_objects.py

```diff
@@ -47,7 +51,7 @@ class EstimatorWithoutFit(object):
     pass
 
 
-class EstimatorWithFit(object):
+class EstimatorWithFit(BaseEstimator):
```

Comment:
I thought that cross_val_score / clone would try to work (with some fallback to `getattr / `setattr`) if the estimator instance does not implement `get_params` / `set_params`. But fair enough.